examples/transpiler/BUILD.bazel

```diff
@@ -31,7 +40,7 @@ ts_project(
 )
 
 # Runs babel to transpile ts -> js
-# and does not produce any declaration outputs.
+# and does not produce any declaration outputs due to noEmit=True.
 # `tsc` is used for type-check only, as a validation action
 # (run bazel with --norun_validations to skip typechecking)
```

Comment:
Setting --norun_validations is no longer correct either after https://github.com/aspect-build/rules_ts/pull/690 right?